chart/values.yaml

```diff
@@ -65,6 +76,12 @@ ingress:
   hostname: ''
   tls: []
 
+# -- (map) customization options, see https://github.com/Luzifer/ots/wiki/Customization
+customizations: 
+  metricsAllowedSubnets: 
+    # default to Pod IPs
+    - 10.0.0.0/24
```

Comment:
That's too strict. The usual k8s setup according to the book uses `10.244.0.0/16`. This maybe should use `10.0.0.0/8`.